user_groups/user_groups.md

```diff
@@ -311,7 +365,7 @@ POST /v1/usergroups/{userGroupId}/permissions/{resourceId}
 | Name | Type | Description |
 | ------------- |------------- | -------------|
 | userGroupId | `string` | The user group id to request. |
```

Comment:
A user group id to update the permission for.

Reply:
updated